rosdep/python.yaml

```diff
@@ -8876,6 +8876,18 @@ python3-rdflib:
   rhel:
     '8': ['python%{python3_pkgversion}-rdflib']
   ubuntu: [python3-rdflib]
+python3-reportlab:
+  arch: [python-reportlab]
+  debian: [python3-reportlab]
+  fedora: [python3-reportlab]
+  gentoo: [reportlab]
```

Comment:
I think this might be `dev-python/reportlab` at least I see other keys in that style (see python3-requests below this one)

Reply:
You are correct, if I just had checked the links I provided myself 😞 

But it is fixed